kubectl_build/Tiltfile

```diff
@@ -40,6 +40,8 @@ def kubectl_build(ref, context, build_args={}, dockerfile=None,
             kwargs["extra_tag"] = extra_tag
         if cache_from != None:
             kwargs["cache_from"] = cache_from
+        if cache_to != None:
```

Comment:
i don't think this will emit an error? this codepath is building up the args to docker_build, and docker_build does not support cache_to

Reply:
I misunderstood the code but looking closer I agree. This seems to be the fallback logic for making kubectl_build pass through to docker_build if your specific build configuration isn't compatible with kubectl_build. I'll remove this change since my concern is the other direction (I want to add cache_to only for kubectl_build).

Are the CI tests attached here enough to feel confident that my change doesn't break typical use cases? What other kind of testing evidence may be needed for this PR to be merged?

Reply:
ya, this is fine. there are tests but they're more for "make sure the basic use-cases work end to end" rather than for "test every combination of flags"